ui-tests/src/test/java/org/openmrs/reference/page/PatientVisitsDashboardPage.java

```diff
@@ -44,6 +49,22 @@ public PatientVisitsDashboardPage(Page parent, WebElement waitForStaleness) {
     public String getPageUrl() {
         return "coreapps/patientdashboard/patientDashboard.page";
     }
+    
+    public void goToEditVitalsPage() {
+        findElement(EDIT_VITALS_ICON).click();
+    }
+    
+    public void goToViewVitalsPage() {
+        findElement(VIEW_VITALS_ICON).click();
+    }
+    
+    public void deleteVitals() {
+        findElement(DELETE_VITALS_ICON).click();
+    }
+    
+    public void confirmDeleteVitals() {
+        findElement(COMFIRM_DELETE_BTN).click();
+    }
```

Comment:
We can rather use clickOn(elementName);  than findElement().click. Having something like 

```js
public void deleteVitals() {
        clickOn(DELETE_VITALS_ICON);
    }

```

Reply:
@kdaud does this look better ?

Reply:
@Irenyak1 did you take a look at the third comment [above](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/562#discussion_r709296827)?

Reply:
> We can rather use `clickOn(elementName) ` than findElement().click. Having something like
> 
> ```js
> public void goToEditVitalsPage() {
>     clickOn(EDIT_VITALS_ICON);
>     }
> ```

@kdaud should I do that for all the methods in that class even those that were already existing or I only change what I have introduced?

Reply:
> or I only change what I have introduced?

Yes